laterpay/application/Controller/Base.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * LaterPay base controller.

```

Comment:
Actually base controller do nothing... it's just a layer between view and end controllers that does common things such as logger initialization and etc.